doc/src/vpr/command_line_usage.rst

```diff
@@ -669,6 +669,46 @@ If any of init_t, exit_t or alpha_t is specified, the user schedule, with a fixe
 
     **Default:** ``0.8``
 
+.. option:: --alpha_min <float>
+
+    The minimum (starting) update factor (alpha) used.
+    Ranges between 0 and alpha_max.
+    Setting this flag selects Dusty's annealing schedule.
```

Comment:
"Dusty's annealing schedule" should probably link to the documentation for the new annealing schedule?

Reply:
Done.